packages/visx-demo/package.json

```diff
@@ -66,7 +66,7 @@
     "@visx/text": "2.17.0",
     "@visx/threshold": "2.18.0",
     "@visx/tooltip": "2.17.0",
-    "@visx/visx": "2.16.0",
+    "@visx/visx": "2.18.0",
```

Comment:
this was not the latest, which meant a copy of all `@visx/*` packages were in a local `node_modules` and hid some errors.